src/main/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BATTLESHIP</title>
+    <link rel="stylesheet" href="../../src/css/main.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <main class="wrapper__main">
+            <img class="logo" src="#" alt="ship" />
+            <h1 class="title">BATTLESHIP</h1>
+
+            <nav class="menu">
+                <ul class="menu__items">
+                    <li class="menu__item">
+                        <a href="gameboard/game.html" class="menu__item_link">Start Game</a><span class="line"></span>
+                    </li>
+                    <li class="menu__item">
+                        <a href="howtoplay/howtoplay.html" class="menu__item_link">How to Play</a><span
```

Comment:
If you add the `<span class="link">` to all links, wouldn't it be easier to style the class `menu__item_link` to have border-bottom 😄 and remove `<span>` tags?